client/src/components/layouts/global.css

```diff
@@ -227,11 +227,9 @@ input[type='submit'],
   white-space: pre-line;
 }
 
-button:hover,
+button:hover:not([aria-disabled='true'], [role='menuitem'], .map-title),
```

Comment:
This was overriding the tailwind styles, you will need to use class instead of the `:not` to reduce complexity, but this is a quick patch for the issue